src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -42,6 +42,7 @@ public class JsonAdaptedPersonTest {
     private static final String VALID_DIAGNOSIS = BENSON.getDiagnosis().toString();
     private static final String VALID_MEDICATION = BENSON.getMedication().toString();
     private static final String VALID_NOTES = BENSON.getNotes().toString();
+    private static final JsonAdaptedAppointment VALID_APPOINTMENT = new JsonAdaptedAppointment(BENSON.getAppointment());
```

Comment:
I think Benson's appointment here is null as TypicalPersons file is not updated yet, do continue to add more test related files in the future as an issue 🥂